libs/shared/src/entities/messages/messages.interface.ts

```diff
@@ -28,10 +28,12 @@ export interface IMessage {
   subject?: string;
 }
 
+type UrlTarget = '_self' | '_blank' | '_parent' | '_top' | '_unfencedTop';
```

Comment:
Could we export this `UrlTarget` type and reuse it in other places within `@novu/shared` and `@novu/api`?